cmd/autogen.sh

```diff
@@ -27,6 +27,9 @@ case "$ID" in
 	ubuntu)
 		extra_opts="--libexecdir=/usr/lib/snapd --enable-nvidia-ubuntu"
 		;;
+	fedora)
+		extra_opts="--disable-apparmor"
```

Comment:
Will it automatically use `/usr/libexec/snapd` for the libexecdir? If not, you should probably set it. Don't forget that we have quite a few params to set here.

See: http://pkgs.fedoraproject.org/cgit/rpms/snap-confine.git/tree/snap-confine.spec#n38